backend/auth_backends.py

```diff
@@ -18,10 +18,5 @@ def authenticate(self, request, username=None, password=None, **kwargs):
             return None
         else:
             if user.check_password(password):
-                if user.is_active:
-                    return user
-
-                if user.awaiting_email_verification:
```

Comment:
This was going to initially be used to actually send the user to a different page if still awaiting email verification (or at least send a notification), but that's my fault for not adding a comment or anything. I'll just write myself a note to add this later